packages/gafl-webapp-service/src/pages/multibuy/change-licence-options/__tests__/route.spec.js

```diff
@@ -0,0 +1,391 @@
+          cache: () => ({
+            helpers: {
+              status: { getCurrentPermission: () => mockStatus },
+              transaction: { getCurrentPermission: () => mockTransaction }
+            }
+          })
+        }
+        const mockStatus = mockStatusCacheGet.mockImplementationOnce(() => ({ renewal: false }))
+        const mockTransaction = mockTransactionCacheGet.mockImplementationOnce(() => ({
+          licenceStartDate: mockStatus.renewal ? RENEWAL_START_DATE.uri : LICENCE_TO_START.uri,
+          licensee: { firstName: 'John', lastName: 'Smith', birthDate: '1996-01-01' },
+          licenceLength: '12M',
+          permit: {
+            cost: 6
+          },
+          concessions: {
+            concession,
+            find: jest.fn(c => {
+              return c.type === 'disabled'
+            })
+          },
+          licenceType: type,
+          numberOfRods: numOfRods
+        }))
+        const result = await getData(mockRequest(request))
+        expect(result.licenceTypeStr).toBe(expectedResult)
+      }
+    )
+
+    it('should return isContinuing as true if renewedEndDate is equal to licenceStartDate', async () => {
+      const request = {
+        cache: () => ({
+          helpers: {
+            status: { getCurrentPermission: () => mockStatus },
+            transaction: { getCurrentPermission: () => mockTransaction }
+          }
+        })
+      }
+      const mockStatus = mockStatusCacheGet.mockImplementationOnce(() => ({ renewal: true }))
+      const mockTransaction = mockTransactionCacheGet.mockImplementationOnce(() => ({
+        licensee: { firstName: 'John', lastName: 'Smith', birthDate: '1996-01-01' },
+        permit: {
+          cost: 6
+        },
+        renewedEndDate: '2022-06-22T14:48:00.000Z',
+        licenceStartDate: '2022-06-22T14:48:00.000Z'
+      }))
+      const result = await getData(mockRequest(request))
+      expect(result.isContinuing).toBeTruthy()
+    })
+
+    it('should return isContinuing as false if both renewedEndDate is not equal to licenceStartDate', async () => {
+      const request = {
+        cache: () => ({
+          helpers: {
+            status: { getCurrentPermission: () => mockStatus },
+            transaction: { getCurrentPermission: () => mockTransaction }
+          }
+        })
+      }
+      const mockStatus = mockStatusCacheGet.mockImplementationOnce(() => ({ renewal: true }))
+      const mockTransaction = mockTransactionCacheGet.mockImplementationOnce(() => ({
+        licensee: { firstName: 'John', lastName: 'Smith', birthDate: '1996-01-01' },
+        permit: {
+          cost: 6
+        },
+        renewedEndDate: '2022-06-22T14:48:00.000Z',
+        licenceStartDate: '2022-07-22T15:48:00.000Z'
+      }))
+      const result = await getData(mockRequest(request))
+      expect(result.isContinuing).toBeFalsy()
+    })
+
+    it('should return true if a concession has been selected and only if disabled', async () => {
```

Comment:
This needs a slightly better description: 'should return disabled flag as true if disabled concession is set'